.github/workflows/builds.yml

```diff
@@ -8,33 +8,41 @@ on:
 
 env:
   CTEST_OUTPUT_ON_FAILURE: 1
-  
+
 jobs:
   builds:
     name: ${{ matrix.platform.name }}-${{ matrix.build }}
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2004:v11
+    container: ${{ matrix.platform.container }}
     strategy:
       matrix:
         platform:
           - name: CPU
             container: ghcr.io/acts-project/ubuntu2004:v11
           - name: CUDA
-            container: ghcr.io/acts-project/ubuntu1804_cuda:v11
+            container: ghcr.io/acts-project/ubuntu2004_cuda:v13
         build:
           - Release
           - Debug
     steps:
       - name: Dependencies
         run: apt-get install -y git-lfs
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1      
         with:
           submodules: true
           lfs: true
       - name: Configure
-        run: cmake -DCMAKE_BUILD_TYPE=${{ matrix.build }} -B build -S .
+        run: >
```

Comment:
I don't think it is ideal to do this with bash scripting, but for now it's fine as it is. Let's think of a more robust way to do this at a later point.